tests/lib/schema_validation_test.py

```diff
@@ -65,7 +65,22 @@
                 'out-discards': 0,
                 'out-errors': 0,
             },
-        }
+        },
+        {
+            'name': 'br0',
+            'state': 'up',
+            'type': 'linux-bridge',
+            'bridge': {
+                'options': {
+                    'vlan-filtering': True,
+                    'vlans': [
+                        {'vlan-range-min': 100},
+                        {'vlan-range-min': 100, 'vlan-range-max': 200},
```

Comment:
Seems a overlapping here on VLAN 100.

Reply:
> If possible, please elaborate the use case for this schema for me to better understand it.
> 
> Thanks.

What kubevirt needs is currently quite simple:
- enable vlan_filtering
- attach a host NIC/bond in trunk mode
- accept it when command-line tools attach a port in trunk or a single
vlan mode.